docs/troubleshooting/monitoring.md

```diff
@@ -69,3 +69,121 @@ rancher-monitoring-grafana-d9c56d79b-cp86w               3/3     Running   0
+
+# kubectl get volume -A
+NAMESPACE         NAME                                       STATE      ROBUSTNESS   SCHEDULED   SIZE          NODE     AGE
+longhorn-system   pvc-1b2fbbe9-14b1-4a65-941a-7d5645a89977   attached   degraded                 5368709120    harv31   43h
+longhorn-system   pvc-7c6dcb61-51a9-4a38-b4c5-acaa11788978   attached   degraded                 53687091200   harv31   43h
+longhorn-system   pvc-b2b2c07c-f7cd-4965-90e6-ac3319597bf7   attached   degraded                 2147483648    harv31   43h
+```
+
+### Scale Down a Deployment
+
+To detach the `Volume`, you need to scale down the `deployment` which uses the `Volume`.
+
+The example below is against the PVC claimed by `rancher-monitoring-grafana`.
+
+Find out the `deployment` in the namespace `cattle-monitoring-system`.
+
+```
+# kubectl get deployment -n cattle-monitoring-system
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+rancher-monitoring-grafana              1/1     1            1           43h  // target deployment
+rancher-monitoring-kube-state-metrics   1/1     1            1           43h
+rancher-monitoring-operator             1/1     1            1           43h
+rancher-monitoring-prometheus-adapter   1/1     1            1           43h
+```
+
+Scale down the deployment `rancher-monitoring-grafana` to 0.
+
+```
+# kubectl scale --replicas=0 deployment/rancher-monitoring-grafana  -n cattle-monitoring-system
+```
+
+Check the deployment and the volume.
+
+```
+# kubectl get deployment -n cattle-monitoring-system
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+rancher-monitoring-grafana              0/0     0            0           43h  // scaled down
+rancher-monitoring-kube-state-metrics   1/1     1            1           43h
+rancher-monitoring-operator             1/1     1            1           43h
+rancher-monitoring-prometheus-adapter   1/1     1            1           43h
+
+# kubectl get volume -A
+NAMESPACE         NAME                                       STATE      ROBUSTNESS   SCHEDULED   SIZE          NODE     AGE
+longhorn-system   pvc-1b2fbbe9-14b1-4a65-941a-7d5645a89977   attached   degraded                 5368709120    harv31   43h
+longhorn-system   pvc-7c6dcb61-51a9-4a38-b4c5-acaa11788978   attached   degraded                 53687091200   harv31   43h
+longhorn-system   pvc-b2b2c07c-f7cd-4965-90e6-ac3319597bf7   detached   unknown                  2147483648             43h  // volume is detached
+```
+
+### Expand Volume
+
+In Longhorn WebUI, the related volume becomes `Detached`. Click the icon in the `Operation` colume, and select `Expand Volume`.
+
+![](/img/v1.1/troubleshooting/4-select-volume-to-expand.png)
+
+Input a new size, and `Longhorn` will expand the volume to this size.
+
+![](/img/v1.1/troubleshooting/5-expand-volue-to-new-size.png)
+
+### Scale Up a Deployment
+
+To scale up a deployment to the original replicas:
```

Comment:
`to the original replicas` needs to be more detailed as mentioned before

Reply:
tried to add more details, please check again.

BTW: A common understanding is that the `trouble-shooting` is targeting those technical users, when they have certain background of `kubernetes`, they will not be confused by `replica` concept.

Reply:
Can we say: To scale up a deployment to `1`:
I remember suggesting it in yesterday's comment, but I'm not sure if my understanding is correct

Reply:
The `scale up` is the reverse operation of `scale down`, it needs to keep the number of `replicas` as the original values. We use `the original replicas` to highlight this key point.

Please take a look of the updated description, thanks.